app/lib/timetable/timetable_page/school_class_filter/school_class_filter.dart

```diff
@@ -119,28 +141,41 @@ class _SchoolClassFilterBottomBarState
     );
   }
 
+  PopupMenuItem<SchoolClassFilter> schoolClassToItem(
```

Comment:
Nit: You don't have to change it
```suggestion
  PopupMenuItem<SchoolClassFilter> schoolClassToFilterItem(
```